metrics/report/main.go

```diff
@@ -0,0 +1,88 @@
+/// Copyight (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"github.com/BurntSushi/toml"
+	"log"
+	"os/exec"
+	"strings"
+)
+
+// Configuration stores all necessary
+// information for checktool execution
+// and sent output as a report by email.
+type Configuration struct {
+	Mail MailConf
+	Ck   Checktool `toml:"checktool"`
+}
+
+// MailConf stores the authentication
+// configuration for sending the report
+// by email. This information is parsed
+// from TOML input file.
+type MailConf struct {
+	Smtp    string   // SMTP server address
```

Comment:
sure thing - the script invokes a number of tools (gocyclo, misspell, golint, ineffassign, vet, fmt). And note, it runs `fmt` with the -s 'simplify' option as well. Heh heh heh...